ambry-rest/src/main/java/com/github/ambry/rest/PublicAccessLogHandler.java

```diff
@@ -158,6 +163,7 @@ private void logHeaders(String tag, HttpMessage message, String[] headers) {
    */
   private void logDurations() {
     long nowMs = System.currentTimeMillis();
+    logMessage.append("bytes sent=").append(responseBytesSent).append(", ");
```

Comment:
for some reason, this section does not separate using commas, only spaces, so we can remove the comma to keep it consistent.

In the long run, we should probably take another look at the log message format to make it easier to parse programatically.

Reply:
sure